src/functions/unreferencedReusableObject.ts

```diff
@@ -0,0 +1,28 @@
+import { IFunction } from '../types';
+
+export type OasReusableObjectsLocation = '#/definitions' | '#/components/schemas';
+
+export const unreferencedReusableObject: IFunction<{ reusableObjectsLocation: OasReusableObjectsLocation }> = (
+  data,
+  opts,
+  _paths,
+  otherValues,
+) => {
+  if (typeof data !== 'object' || data === null) return [];
```

Comment:
Let's use `isObject` from `src/utils/isObject.ts`

Reply:
Sure? The doco says 
```
/** @deprecated since v4.0.0 - use `value !== null && typeof value === 'object'` instead. */
```

Reply:
Hm, are you importing the right file? I don't remind us deprecating that.
Perhaps VS Code insert an import for you and it points to a different file?
This is the function we want to impor thttps://github.com/stoplightio/spectral/blob/develop/src/utils/isObject.ts

Reply:
Duh. I wrongfully initially imported node 'util' (which is indeed deprecated). Thanks for having steered me back in the proper direction.

Fixed